BeeTracker/app/src/main/res/layout/capture_fragment.xml

```diff
@@ -38,9 +37,10 @@
 
     <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="66dp"
+        android:layout_height="80dp"
```

Comment:
dimens.xml

Reply:
i don't think this needs to go in `dimens.xml` because this value will not be reused anywhere else